client/js/middleware/serialization/qbank/serializers/survey.jsx

```diff
@@ -0,0 +1,78 @@
+import _              from 'lodash';
+import baseSerializer from './base';
+import { scrub }      from '../../serializer_utils';
+import genusTypes     from '../../../../constants/genus_types';
+import guid           from '../../../../utils/guid';
+
+function serializeChoices(originalChoices, newChoiceAttributes) {
+  const choices = _.map(originalChoices, (choice) => {
+    const updateValues = newChoiceAttributes[choice.id];
+    return {
+      id: choice.id,
+      text: _.get(updateValues, 'text') || choice.text,
+      order: _.get(updateValues, 'order') || choice.order,
+      delete: _.get(updateValues, 'delete'),
+    };
+  });
+
+  if (newChoiceAttributes.new) {
+    choices.push({
+      id: guid(),
+      text: '',
+      order: choices.length,
+    });
+  }
+
+  return choices;
+}
+
+function serializeQuestion(originalQuestion, newQuestionAttributes) {
+  const newQuestion = {
+    id: originalQuestion.id,
+    genusTypeId: newQuestionAttributes.type === 'reflection' ? genusTypes.question.reflection : genusTypes.question.multipleReflection,
+    questionString: newQuestionAttributes.text,
+    multiAnswer: newQuestionAttributes.multiAnswer,
+    shuffle: newQuestionAttributes.shuffle,
+    timeValue: newQuestionAttributes.timeValue,
+    fileIds: {},
+    choices: null,
+  };
+
+  if (newQuestionAttributes.choices) {
+    newQuestion.choices = serializeChoices(originalQuestion.choices, newQuestionAttributes.choices);
+  }
+
+  return scrub(newQuestion);
+}
+
+function serializeAnswers(originalChoices, newChoiceAttributes) {
+  let newFeedback = null;
+  _.forEach(newChoiceAttributes, (choice) => {
```

Comment:
All the feedback need to be the same, so we go through all the choices, and if one of them has feedback, all of them need that same feedback. I could do `const thing = _.find(newChoiceAttributes, choice => !!choice.feedback);`  `const newFeedBack = thing[0].feedback;` Either way I guess